client/my-sites/plugins/plugin-management-v2/types.ts

```diff
@@ -81,3 +67,10 @@ export type PluginActionStatusMessage = {
 		[ PLUGIN_INSTALLATION_UP_TO_DATE ]?: ReactNode;
 	};
 };
+
+export type ExtendedPlugin = Plugin & {
```

Comment:
I added a comment but I can't think of a better name because these are just random properties that are stuck on to the plugin objects here and there in the component code, so there's not a new specific name to give it. I consider this an anti-pattern but it would be a lot of work to change it.

Reply:
Maybe something around `PluginComponentProps` then?